### PR TITLE
Minor bugfix for 6.1 Service Fabric data plane spec

### PR DESCRIPTION
**Pull request by samedder**
Changes include
- Adding 6.1 tag to test auto python SDK generation
- Docstring fixes to 6.1 Service Fabric data plane spec

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/servicefabric/data-plane/readme.md [hunk 1]
```diff
@@ -60,6 +60,15 @@ input-file:
 ```
 
 
+### Tag: 6.1
```
Comment (lmazuel) [inline specification/servicefabric/data-plane/readme.md]: Please add this to Python conf (or what Python version you want actually)
```
package-version: 6.1.2.9
```

Comment (samedder) [reply specification/servicefabric/data-plane/readme.md]: Can we have the python package match the spec version? So in this case 6.1.2.9?

Comment (samedder) [reply specification/servicefabric/data-plane/readme.md]: It looks like it does match already, I think that's okay

**Comment by samedder** [comment]
@sergey-shandar can we get this merged today?

**Comment by sergey-shandar** [comment]
@lmazuel Are ok with the changes?

File: specification/servicefabric/data-plane/readme.md (hunk 1 above)
Comment (lmazuel) [reply specification/servicefabric/data-plane/readme.md]: @samedder It was longer to explain than to do it, so I pushed a commit in your PR :)

**Comment by lmazuel** [comment]
@sergey-shandar Ok for me, I pushed the needed Python conf

**Comment by lmazuel** [comment]
@marstr your batch conf for Go is incorrect, based on the errors I see.

File: specification/servicefabric/data-plane/readme.md (hunk 1 above)
Comment (samedder) [reply specification/servicefabric/data-plane/readme.md]: I see, is there any issue with using `6.1.2` instead of `6.1.2.9` ?

**Comment by samedder** [comment]
@lmazuel I don't mind ignoring the version for now, whatever you think is the best we'll follow, we just need to get this merged as soon as possible.

**Comment by lmazuel** [comment]
@marstr Actually, autorest used the version 3.x of the Go generator, which was not planned. @olydis will try to find why. In the meantime, we might change "preview" by "^2" (or whatever good npm syntax).

**Comment by samedder** [comment]
@lmazuel can we merge this without the Go generator fixes?

**Comment by lmazuel** [comment]
@samedder Yes. @sergey-shandar please merge at your convenience when you're happy with it. Python package is already released.

**Review by sergey-shandar — APPROVED** [review]

**Pull request merged**